planning/behavior_path_planner/autoware_behavior_path_lane_change_module/src/scene.cpp

```diff
@@ -1614,8 +1631,9 @@ LaneChangePath NormalLaneChange::get_candidate_path(
 
 bool NormalLaneChange::check_candidate_path_safety(
   const LaneChangePath & candidate_path, const lane_change::TargetObjects & target_objects,
-  const double lane_change_buffer, const bool is_stuck) const
+  const bool is_stuck) const
 {
+  const auto & lane_change_buffer = common_data_ptr_->transient_data.current_dist_buffer.min;
```

Comment:
```suggestion
  const auto lane_change_buffer = common_data_ptr_->transient_data.current_dist_buffer.min;
```
trivial type, so reference is not needed.